ush/generate_FV3LAM_wflow.py

```diff
@@ -1043,146 +998,82 @@ def generate_FV3LAM_wflow():
             */{CRON_RELAUNCH_INTVL_MNTS} * * * * cd {EXPTDIR} && ./launch_FV3LAM_wflow.sh called_from_cron=\"TRUE\"
             """
         )
-    #
-    # If necessary, run the NOMADS script to source external model data.
-    #
-    if NOMADS:
-        print("Getting NOMADS online data")
-        print(f"NOMADS_file_type= {NOMADS_file_type}")
-        cd_vrfy(EXPTDIR)
-        NOMADS_script = os.path.join(USHdir, "NOMADS_get_extrn_mdl_files.h")
-        run_command(
-            f"""{NOMADS_script} {date_to_str(DATE_FIRST_CYCL,format="%Y%m%d")} \
-                      {date_to_str(DATE_FIRST_CYCL,format="%H")} {NOMADS_file_type} {FCST_LEN_HRS} {LBC_SPEC_INTVL_HRS}"""
-        )
 
+    # If we got to this point everything was successful: move the log file to the experiment directory.
+    mv_vrfy(logfile, EXPTDIR)
+
+def get_nomads_data(NOMADS_file_type,EXPTDIR,USHdir,DATE_FIRST_CYCL,CYCL_HRS,FCST_LEN_HRS,LBC_SPEC_INTVL_HRS):
+    print("Getting NOMADS online data")
+    print(f"NOMADS_file_type= {NOMADS_file_type}")
+    cd_vrfy(EXPTDIR)
+    NOMADS_script = os.path.join(USHdir, "NOMADS_get_extrn_mdl_files.sh")
+    # run_command(f"""{NOMADS_script} {date_to_str(DATE_FIRST_CYCL,format="%Y%m%d")} \
+    #                 {date_to_str(DATE_FIRST_CYCL,format="%H")} {NOMADS_file_type} {FCST_LEN_HRS} {LBC_SPEC_INTVL_HRS}""")
+    raise Exception("Nomads script does not work")
+
+def setup_logging(logfile: str = 'log.generate_FV3LAM_wflow') -> None:
+    """
+    Sets up logging, printing high-priority (INFO and higher) messages to screen, and printing all
+    messages with detailed timing and routine info in the specified text file.
+    """
+    logging.basicConfig(level=logging.DEBUG,
+                        format='%(name)-22s %(levelname)-8s %(message)s',
+                        filename=logfile,
+                        filemode='w')
+    logging.debug(f'Finished setting up debug file logging in {logfile}')
+    console = logging.StreamHandler()
+    console.setLevel(logging.INFO)
```

Comment:
No, all messages of "INFO" level or higher are printed to screen; this includes ERROR and WARNING messages as well. 

I hope to later do a sanitation of the workflow to demote some messages to "DEBUG" level, and have those only print to the log file unless the user specifies otherwise. That is for future work though.

Reply:
Thanks for the clarification!